### DISCUSSION TITLE
Just got an attack on my pocketbase api server

### DISCUSSION THREAD

**Discussion by GD-coding** [pocketbase/pocketbase#3208] [Q&A]
Is there some sort of vulnerability with this?

![image](https://github.com/pocketbase/pocketbase/assets/63622794/62688862-86d6-4b26-bdaa-4aed7eb5fc54)

**Comment by ganigeorgiev** [comment]
It seems like some automated script is trying to find an accidentally published `.git` dir on your server. You can ignore them or ban the IP with ufw, iptables, etc.
